diff --git a/Directory.Build.props b/Directory.Build.props
index 88b63d156..38f31576e 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -50,16 +50,11 @@
     This section is for getting & setting the gitHash value, which can easily be accessed
     via the Core.Utilities.AssemblyHelpers class.
   -->
-  <Target Name="SetSourceRevisionId" BeforeTargets="CoreGenerateAssemblyInfo">
-    <Exec Command="git describe --long --always --dirty --exclude=* --abbrev=8" ConsoleToMSBuild="True" IgnoreExitCode="False">
-      <Output PropertyName="SourceRevisionId" TaskParameter="ConsoleOutput"/>
-    </Exec>
-  </Target>
-  <Target Name="WriteRevision" AfterTargets="SetSourceRevisionId">
+  <Target Name="WriteRevision">
     <ItemGroup>
       <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
         <_Parameter1>GitHash</_Parameter1>
-        <_Parameter2>$(SourceRevisionId)</_Parameter2>
+        <_Parameter2>debian</_Parameter2>
       </AssemblyAttribute>
     </ItemGroup>
   </Target>
